src/box/raft.c

```diff
@@ -45,6 +46,7 @@ struct raft box_raft_global = {
 };
 
 enum election_mode box_election_mode = ELECTION_MODE_INVALID;
+bool election_fencing;
```

Comment:
Let's initialize it statically, not in `box_raft_init`.
We usually do so for all cfg options.

Reply:
Fixed